### PR TITLE
Update navigator layers

### LINKED ISSUE #65: Update layers to Navigator latest version
You get the following message when using a layer file from the `attack-control-framework-mappings` on the navigator.

image

### PR DESCRIPTION
**Pull request by emmanvg**
## Description of what has changed
The layer files for both NIST 800-53 R4 and R5 have been updated with the newest `attack-navigator` release `4.3`. It should resolve the prompt received when using layer files from this repository. @isaisabel 

## Issues addressed by pull request
Closes #65

### CODE REVIEW

**Comment by emmanvg** [comment]
I did not update the ATT&CK version to "9" since I was not sure if that would affect the current mappings. When testing this change you would get the following dialog box when loading a layer from this repo.

image

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,24 +14,7 @@
...
-- Fixed broken regex which was leading to erroneous mappings, in particular mappings to control enhancements. See issue [#56](https://github.com/center-for-threat-informed-defense/attack-control-framework-mappings/issues/56).
-
-# 15 December 2020
-## nist800-53-r4 v1.0 and nist800-53-r5 v1.0
-- Initial release of security control framework mapping methodology and tools. 
-- Initial release of NIST 800-53 R4 mappings to ATT&CK version 8.1.
-- Initial release of NIST 800-53 R5 mappings to ATT&CK version 8.1.
\ No newline at end of file
+- NIST 800-53 R4 layers updated with attack-navigator 4.3.
+- NIST 800-53 R5 layers updated with attack-navigator 4.3.
```
Comment (isaisabel) [inline CHANGELOG.md]: Updates to the changelog should not overwrite the previous contents of the changelog, rather the new update should be prepended to the changelog.

**Review by isaisabel — CHANGES_REQUESTED** [review]
Changes to layer files looks good, but changelog was not handled properly and we should also change [mappingsToHeatmaps](https://github.com/center-for-threat-informed-defense/attack-control-framework-mappings/blob/master/util/mappingsToHeatmaps.py#L38-L39) so that future generated layers have the right version numbers as well.

**Comment by isaisabel** [comment]
> I did not update the ATT&CK version to "9" since I was not sure if that would affect the current mappings. When testing this change you would get the following dialog box when loading a layer from this repo.
> 
> image

We've not upgraded to ATT&CK version 9 in these mappings (they still map to v8) so the ATT&CK version referenced in the layer should not change.

File: CHANGELOG.md (hunk 1 above)
Comment (emmanvg) [reply CHANGELOG.md]: Good points, I reverted the CHANGELOG and updated the mappingsToHeatmaps.py script values

**Comment by emmanvg** [comment]
> We've not upgraded to ATT&CK version 9 in these mappings (they still map to v8) so the ATT&CK version referenced in the layer should not change.

Perfect, Good to know!

**Review by isaisabel — APPROVED** [review]

**Pull request merged**